minard/templates/noise.html

```diff
@@ -23,8 +25,12 @@
                     <td>{{ "%.0f" % (run["average_noiserate"]|float) }}</td>
 		    {% if "average_qhl_hhp" in run %}
 		    <td>{{ "%.1f" % (run["average_qhl_hhp"]|float) }}</td>
-		    {% else %}
-		    <td>{{ "ND" }}</td>
+		    {% endif %}
+		    {% if "n_pgt" in run %}
+		    <td>{{ "%.0f" % (run["n_pgt"]|float) }}</td>
```

Comment:
I think you probably want the <td> blocks there regardless of whether the column is present. Otherwise, if the n_pgt column is missing from one row but online_pmt is not, it will show up in the wrong column.